### ISSUE TITLE
[nssdca-delivery] urn:nasa:pds:dart_teleobs::1.0

### ISSUE THREAD

**Issue by tbarnes4** [NASA-PDS/operations#398]
### Discipline Node Information

* **Delivering Node**: SBN-UMD

* **NSSDCA Delivery Package**: https://pdssbn.astro.umd.edu/holdings/pds4-dart_teleobs-v1.0/SUPPORT/NSSDCA/

* **Validation report**: https://pdssbn.astro.umd.edu/holdings/pds4-dart_teleobs-v1.0/SUPPORT/NSSDCA/dart_teleobs_v1.0_20230517_validate_rpt.txt

----

### Engineering Node Process

See the internal EN process at https://pds-engineering.jpl.nasa.gov/content/nssdca_interface_process

**Comment by c-suh** [comment]
@tbarnes4 this set has been posted for NSSDCA processing. From tomorrow, you can check the status at https://nssdc.gsfc.nasa.gov/psi/ReportPDS4.jsp using the SIP LID below:

SIP LID:
* urn:nasa:pds:system_bundle:product_sip_deep_archive:dart_teleobs_v1.0_20230517

**Comment by tbarnes4** [comment]
@c-suh I noticed that these are now listed as "Archived" at the NSSDCA 10 days ago. Is there another step or notice you are waiting for?  I know when I process our PDS3 data sets, they give notice when they have accepted responsibility for them and then later give notice when they have been written to tape.

**Comment by smclaughlin7** [comment]
@tbarnes4 For PDS4 a status of "Pre-Ingest" is equivalent to our notice that the NSSDCA has accepted responsibility for archiving a PDS3 dataset, and "Archived" is equivalent our notice that we archived a PDS3 dataset to tape.  Since this submission is "Archived", @c-suh will close out this issue/submission.

@c-suh Atmospheres Node tends to closely monitor their submissions.  They typically close out an issue when the submission is "Archived".  Is it OK if Tilden does this, too, particularly if all collections in a submission are "Archived"?

**Comment by c-suh** [comment]
@smclaughlin7 yes, it is alright (and appreciated) if @tbarnes4 closes out any tickets where all submissions are "Archived". Thank you for asking!
